src/hotspot/share/gc/g1/g1MMUTracker.cpp

```diff
@@ -56,9 +56,9 @@ void G1MMUTracker::remove_expired_entries(double current_time) {
   guarantee(_no_entries == 0, "should have no entries in the array");
 }
 
-double G1MMUTracker::calculate_gc_time(double current_time) {
+double G1MMUTracker::calculate_gc_time(double time_stamp) {
```

Comment:
I think `current_timestamp` reads better; in this file/context `X_time` is (often) used to mean a duration. Keeping the `current` prefix helps also; from the callee's perspective, the arg represents "now".